easybuild/easyconfigs/p/PyTables/PyTables-3.8.0-foss-2023a.eb

```diff
@@ -47,11 +47,11 @@ exts_list = [
     }),
     ('tables', version, {
         'patches': [
-            'PyTables-%(version)s_fix-libs.patch',
+            'PyTables-3.8.0_fix-libs.patch',
```

Comment:
No need for these changes

Reply:
Yeah, no idea what happened there, maybe a merge of a similar version upstream.